src/Input/Command.php

```diff
@@ -298,9 +298,9 @@ public function showHelp(): mixed
         $io     = $this->io();
         $helper = new OutputHelper($io->writer());
 
-        $io->bold("Command {$this->_name}, version {$this->_version}", true)->eol();
-        $io->comment($this->_desc, true)->eol();
-        $io->bold('Usage: ')->yellow("{$this->_name} [OPTIONS...] [ARGUMENTS...]", true);
+        $io->help_header("Command {$this->_name}, version {$this->_version}", true)->eol();
```

Comment:
please add all new magic methods into docblock definition of Interactor class

Reply:
Done!